packages/cli/src/commands/alter.ts

```diff
@@ -315,7 +324,9 @@ export async function alter(
     }
   }
 
-  await resolver.publish([fullPackageRef], chainId, subpkgUrl, metaUrl || '');
+  if (!noPublish) {
```

Comment:
ooc why was `noPublish` option added to this command? is it getting in the way of something?

Reply:
I was needing it to be able to test it out more easily, without it trying to publish to the registry the result.